pkg/sqlite/query_test.go

```diff
@@ -0,0 +1,289 @@
+package sqlite_test
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/operator-framework/operator-registry/pkg/api"
+	"github.com/operator-framework/operator-registry/pkg/sqlite"
+	"github.com/operator-framework/operator-registry/pkg/sqlite/sqlitefakes"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestListBundles(t *testing.T) {
+	const query = `SELECT DISTINCT channel_entry.entry_id, operatorbundle.bundle, operatorbundle.bundlepath,
```

Comment:
Am I misreading something or is this const just not used? Is it injected into something somehow?

Reply:
Nope, good catch. The ListBundles impl calls out to other methods on SQLQuerier, which in turn make other database queries, so I was originally going to set the stub behavior based on the query arg. I ended up stubbing based on invocation count instead and didn't use it.